olog/httpx_client.py

```diff
@@ -134,6 +204,20 @@ async def aget_log(self, id):
         return res.json()
 
     def get_log(self, id):
```

Comment:
make sense

Reply:
I agree with @mrakitin’s suggestion, but as a point of information, “reserved keywords” are things like `if` and `def`. They can never be used as variable names—if you try to use them you’ll get a syntax error. `id` is a “builtin”. You _can_ use it as a variable name, but it can be confusing to readers, and if you later need the built-in `id` in that same scope you’d have to get it via `import builtins; builtins.id`.